approved/README.md

```diff
@@ -1,7 +1,10 @@
-# Revision 4618-ir
+# Approved data models
 
 This distribution contains only those data models that are contained in the amendment to the Implementing Rules for Annex II+III themes, including the updates of the Annex I data themes.
 
+NOTE: Regarding the mapping tables, the original versions in xml have been substituted by the Excel tables currently downloadable using the "[Find Your Scope](https://inspire-regadmin.jrc.ec.europa.eu/dataspecification/FindYourScope.action)" application, more complete than the original ones and easier to update e.g. when a related TG/XML change proposal is endorsed.
```

Comment:
Is "TG/XML change proposal" correct, or should it be "TG/XSD" or "TG/schema"?

Reply:
The proposed changes have been introduced in the main branch.